src/modules/dashboard/components/ColonyHome/ColonyHome.tsx

```diff
@@ -220,21 +239,72 @@ const ColonyHome = ({
     />
   );
 
+  const renderNewTaskButton = (
+    <>
+      {canCreateTask && (
+        <ActionButton
+          button={({ onClick, disabled, loading }) => (
+            <Button
+              appearance={{ theme: 'primary', size: 'medium' }}
+              text={MSG.newTaskButton}
+              disabled={disabled}
+              loading={loading}
+              onClick={throttle(onClick, 2000)}
+            />
+          )}
+          disabled={isInRecoveryMode}
+          error={ActionTypes.TASK_CREATE_ERROR}
+          submit={ActionTypes.TASK_CREATE}
+          success={ActionTypes.TASK_CREATE_SUCCESS}
+          transform={transform}
+          loading={isTaskBeingCreated}
+        />
+      )}
+    </>
+  );
+
+  const crumbs =
+    domains &&
+    domains
+      .sort((a, b) => a.id - b.id)
+      .reduce((accumulator, element) => {
+        if (element && element.name && element.id <= filteredDomainId) {
+          const message = {};
+          message[element.name] = {
+            id: element.name,
+            defaultMessage: element.name,
+          };
+          const messageDescriptor = defineMessages(message);
+          (accumulator as any).push(messageDescriptor);
```

Comment:
Please don't use `as any`. In this case we know the type so you can just do `accumulator as MessageDescriptor[]`.